graphstorm-processing/tests/test_dist_heterogenous_loader.py

```diff
@@ -515,6 +521,46 @@ def create_edges_df_num_label(spark: SparkSession, missing_data_points: int) ->
     return df
 
 
+def create_nodes_df_num_labels(
+    spark: SparkSession, total_data_points=NUM_DATAPOINTS, missing_data_points=0
+) -> DataFrame:
+    """Create a nodes DF with a numeric and a string label for testing.
+
+    Returned DF schema:
+
+        NODE_MAPPING_STR: node_str_ids, unique per node (uuid4),
+        NODE_MAPPING_INT: node_int_ids, unique per node,
+        NUM_LABEL_COL: numeric labels, 0-9 range,
+        STR_LABEL_COL: string labels, 10 random string labels,
+    """
+    node_str_ids = [str(uuid4()) for _ in range(total_data_points)]
+    node_int_ids = np.arange(total_data_points)
+
+    rng = np.random.default_rng(42)
+    # Create random numerical labels with values 0-9
```

Comment:
Do we cover the case for None value?

Reply:
Yes, lines 547-549 set `missing_data_points` number of labels to `None`.